src/ODEmodel.jl

```diff
@@ -28,25 +28,24 @@ function ODEjac(p::AbstractVector{T}, t::Real)::Matrix{T} where {T <: Real}
     A[diagind(A, -1)[Int(nG1 + 3 * nG2 / 4 + 1):(nSp - 1)]] .= p[6]
 
     A[1, nSp] = 2 * p[6]
-
-    lmul!(t, A)
-    return LinearAlgebra.exp!(A)
+    return A
 end
 
-
 """ Find the starting vector from the steady-state of the control condition. """
 function startV(p::AbstractVector{T})::AbstractVector{T} where {T <: Real}
-    v = ones(nSp)
-    A = ODEjac(p, 100.0)
-    u = similar(v)
-
-    for ii = 1:100
-        v /= sum(v)
-        mul!(u, A, v)
-        copyto!(v, u)
-    end
+    @assert all(p .>= 0.0)
+    @assert all(p[7:end] .== 0.0) # No cell death in the control
```

Comment:
@Farnazmdi in the process of adding a more elegant solver for the control condition, I added this assertion that the control condition has no cell death. This is catching during the fitting, possibly indicating an indexing error somewhere. Could you take a look?

Reply:
Of course, will do. Thank you.

Reply:
It seems they are not exactly zero, but close.
![image](https://user-images.githubusercontent.com/34812750/108244221-d2c68c00-7103-11eb-9a28-50807a17df04.png)
I don't know where this happens, but we don't estimate the minimum of cell death, we just set them to zeros here:
![image](https://user-images.githubusercontent.com/34812750/108244414-0c979280-7104-11eb-8113-6d21f4c8554f.png)
Could we assert that they are *close* to zeros?

Reply:
I'm not sure what close would be. If they're evaluated at 0 concentration, and E_control is 0 for death, why aren't they 0?

Reply:
Ahh.. That's why then. The concentration is not zero, because we want to get the log of it, we cannot have absolute zero. I had set the control to be a small number like 0.1. I can fix this right now in my branch so that we can have the concentration of control at zero in this function.

Reply:
Isn't the log of concentration only used for plotting? You should only do that correction before plotting, if so.

Reply:
I will make sure that happens only in plotting.

Reply:
This problem should be fixed.